portability-core/src/main/java/org/dataportabilityproject/cloud/local/InMemoryPersistentKeyValueStore.java

```diff
@@ -20,7 +20,7 @@
 import org.dataportabilityproject.cloud.interfaces.PersistentKeyValueStore;
 
 /** In-memory implementation of backend storage. */
-final class InMemoryPersistentKeyValueStore implements PersistentKeyValueStore {
+public final class InMemoryPersistentKeyValueStore implements PersistentKeyValueStore {
```

Comment:
public or VisibleForTesting?

Reply:
Public, its used across packages for testing.